src/server/honeypot.ts

```diff
@@ -10,27 +10,31 @@ export interface HonetpotConfig {
   randomizeNameFieldName?: boolean;
   nameFieldName?: string;
   validFromFieldName?: string;
-  validFromTimestamp?: number;
   encryptionSeed?: string;
 }
 
 export class SpamError extends Error {}
 
+const DEFAULT_NAME_FIELD_NAME = "name__confirm";
+const DEFAULT_VALID_FROM_FIELD_NAME = "from__confirm";
```

Comment:
These are better default names because I'm almost positive some spam bots avoid filling in fields with the word "honeypot" in them.

These are generic enough to be unlikely to conflict with existing fields, but specific enough to cause bots to try and fill them in.